lib/ddtrace/ext/http.rb

```diff
@@ -6,6 +6,7 @@ module HTTP
       URL = 'http.url'.freeze
       METHOD = 'http.method'.freeze
       STATUS_CODE = 'http.status_code'.freeze
+      ERROR_RANGE = 400...600
```

Comment:
Not sure a 404 is considered an error for most of the users. It really depends by your application and what you're doing with the call